src/EssentialsPE/Events/CreateAPIEvent.php

```diff
@@ -14,32 +14,34 @@ class CreateAPIEvent extends PluginEvent{
     /** @var string */
     private $class;
 
-    /**
-     * @param Loader $plugin
-     * @param BaseAPI::class $api
-     */
-    public function __construct(Loader $plugin, string $api){
+	/**
+	 * CreateAPIEvent constructor.
+	 *
+	 * @param Loader $plugin
+	 * @param string $api
+	 */
+	public function __construct(Loader $plugin, string $api){
```

Comment:
you broke the formatting more here instead of fixing it. The indentation should use spaces, not tabs.

Reply:
Updated PhpStorm settings